### PR TITLE
Allow spaces in names of users or teams

### PR DESCRIPTION
**Pull request by charlottestjohn**

### CODE REVIEW

**Comment by partkyle** [comment]
LGTM

File: src/scripts/pagerduty.coffee [hunk 1]
```diff
@@ -108,10 +108,10 @@ module.exports = (robot) ->
       else
         msg.send "No open incidents"
 
-  robot.respond /(pager|major)( me)? (?:trigger|page) @?([\w\-]+)$/i, (msg) ->
-    msg.reply "Please include a user or schedule to page, like 'hubot pager infrastructure everything is on fire'."
+  robot.respond /(pager|major)( me)? (?:trigger|page) @?([\w\- ]+)$/i, (msg) ->
+    msg.reply "Please include a user or schedule to page, like 'hubot pager infrastructure: everything is on fire'."
```
Comment (partkyle) [inline src/scripts/pagerduty.coffee]: The message does not match what the regex is looking for. Like mentioned in #6, this will already have a user.

**Comment by partkyle** [comment]
This is going to conflict with #6

**Comment by charlottestjohn** [comment]
Will fix the conflict in 6

**Pull request merged**